aether-utils/aether-client/aether/client_test/fixtures/__init__.py

```diff
@@ -29,60 +27,56 @@ def file_to_json(path):
 
 #Projects
 project_name = "TestProject"
-project_def = file_to_json(here+"/fixtures/salad/salad.json")
-project_obj = {
+project_template = {
     "revision": "1",
-    "name": project_name,
-    "salad_schema": project_def,
+    "name": None,
+    "salad_schema": "{}",
     "jsonld_context": "[]",
     "rdf_definition": "[]"
 }
 
 #Schemas
-schema_objs = []
-schema_dir =here+"/fixtures/avro"
-schema_files = os.listdir(schema_dir)
 
+schema_file =here+"/schema/schemas.avro"
```

Comment:
Could you include `flake8` in your tests to enforce a little bit of "styling" in the code? :grimacing:

Reply:
It's in there but it was ignoring the fixtures folder. I'll let it look at the python files and make it ignore the actual artifacts.